lldb/source/API/CMakeLists.txt

```diff
@@ -142,6 +142,7 @@ add_lldb_library(liblldb SHARED ${option_framework}
     lldbValueObject
     lldbVersion
     ${LLDB_ALL_PLUGINS}
+    swiftCore
```

Comment:
I don't see a library named this in `llvm/llvm-project`, did you mean to submit this to Apple's llvm fork?

Which is https://github.com/swiftlang/llvm-project (I think).

Reply:
You are correct, sorry about that. I opened the PR here: https://github.com/llvm/llvm-project/pull/138297